lib/deqm_test_kit/evaluate_measure.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+module DEQMTestKit
+  # tests for $evaluate-measure
+  # rubocop:disable Metrics/ClassLength
+  class EvaluateMeasure < Inferno::TestGroup
+    id 'evaluate_measure'
+    title 'Evaluate Measure'
+    description 'Ensure FHIR server can calculate a measure'
+
+    fhir_client do
+      url :url
+    end
+
+    INVALID_MEASURE_ID = 'INVALID_MEASURE_ID'
+    INVALID_PATIENT_ID = 'INVALID_PATIENT_ID'
+    test do
+      title 'Check $evaluate-measure proper calculation'
+      id 'evaluate-measure-01'
+      description 'Server should properly return a measure report'
+      input :measure_id, :patient_id
+      input :period_start, default: '2019-01-01'
+      input :period_end, default: '2019-12-31'
+
+      run do
+        params = "periodStart=#{period_start}&periodEnd=#{period_end}&subject=#{patient_id}"
+        fhir_operation("/Measure/#{measure_id}/$evaluate-measure?#{params}")
+
+        assert_response_status(200)
+        assert_resource_type(:measure_report)
+        assert_valid_json(response[:body])
```

Comment:
Again, not sure this is in your task scope, but do we want to verify the accuracy of the evaluate-measure results in some way? Do we want to compare against cqf-ruler or anything like that?